stoplight.js

```diff
@@ -0,0 +1,23 @@
+//Turns top red
+$("#red").on("click", function() {
+  $(this).toggleClass("bulb", onOrOff());
```

Comment:
not sure `toggleClass` is being used correctly...in this example, `onOrOff` needs to return a boolean, according to the docs...http://api.jquery.com/toggleclass/